libs/shared/src/lib/components/sidesheet/Header/Header.tsx

```diff
@@ -34,18 +57,23 @@ const StyledRightSection = styled.div`
 `;
 type HeaderProps = {
   title: string;
+  applicationTitle: string;
   onClose: VoidFunction;
 };
 /**
  * Standard component for displaying a sidesheet header.
  * This will add a title and close button to the header.
  */
-export const SidesheetHeader = ({ title, onClose }: HeaderProps) => {
+export const SidesheetHeader = ({ title, applicationTitle, onClose }: HeaderProps) => {

```

Comment:
In addition to applicationTitle, I think  you should add another property to the component:  `color`. An optional prop with the default value of `#8c1159` that is passed down to the styled component `StyledColor`